arch/x86/crc32_fold_pclmulqdq.c

```diff
@@ -597,7 +597,7 @@ uint32_t crc32_pclmulqdq(uint32_t crc32, const unsigned char* buf, uint64_t len)
     /* For lens < 64, crc32_byfour method is faster. The CRC32 instruction for
```

Comment:
I think this comments needs updating to crc32_braid

Reply:
Fixed.